js/packages/cli/src/candy-machine-cli.ts

```diff
@@ -577,14 +577,34 @@ programCommand('update_candy_machine')
     log.info('updated_candy_machine finished', tx);
   });
 
-programCommand('mint_one_token').action(async (directory, cmd) => {
-  const { keypair, env, cacheName } = cmd.opts();
+programCommand('premint_token')
+  .option(
+    '-n --number <number>',
+    'Premint NFTs based on the number entered starting from 0.json or the last minted JSON  if the command has already ran.',
+  )
+  .action(async (directory, cmd) => {
+    const { keypair, env, cacheName, number } = cmd.opts();
 
-  const cacheContent = loadCache(cacheName, env);
-  const configAddress = new PublicKey(cacheContent.program.config);
-  const tx = await mint(keypair, env, configAddress);
+    const cacheContent = loadCache(cacheName, env);
+    const configAddress = new PublicKey(cacheContent.program.config);
```

Comment:
this can be done a little more cleanly. Like if number is not passed in, it just assumes it's 1, and does a loop of one.

Reply:
Yes, if a number isn't passed in it defaults to what mint_one_token did before which was only mint one. To mint more, one would have to use the -n flag. I'll take a look at it.